apps/web/src/lib/components/book-reader/book-reader-header.svelte

```diff
@@ -129,6 +131,16 @@
   </div>
 
   <div class="flex transform-gpu {translateXHeaderFa}">
+    <div
```

Comment:
we reached a point where new header actions may push each other out without a change to the overall structure - for the time being therefore better to add it as merge entry